cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,11 @@
+describe('/App/App.js', () => {
```

Comment:
These are the tests I wrote: navigating to page, seeing the title, seeing movies